.github/workflows/pullrequest.yaml

```
@@ -18,8 +18,15 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runnner.os }}-yarn-${{ hashFiles('**/yarn.lock.json') }}
```

```suggestion
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock.json') }}
```

@flat35hd99 This spell miss is same as the line above.

---

Commit suggestionという機能があるんですね。すごく便利ですね。
間違えてresolveを押してしまったため、片方だけ下のcommitで同様の変更を行っておきました。